spec/board_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'board.rb'
+
+RSpec.describe Board do
+  describe '#winner?' do
+    it 'returns false for a non winning board' do
+      expect(subject.winner?('O')).to eql(false)
+    end
+    it 'returns true for a winning row' do
```

Comment:
kindly leave an empty line between `it` block, check an example below;
#Bad 
```
 it 'should return true when the third row of the board is filled with the same symbol' do
      game.board_game.board = %w[1 2 3 4 5 6 O O O]
      expect(game.win?).to eql(true)    
 end
  it 'should return true when the first column of the board is filled with the same symbol' do
      game.board_game.board = %w[O 2 3 O 5 6 O 8 9]
      expect(game.win?).to eql(true)
  end
```

#Good
```
 it 'should return true when the third row of the board is filled with the same symbol' do
      game.board_game.board = %w[1 2 3 4 5 6 O O O]
      expect(game.win?).to eql(true)
  end

  it 'should return true when the first column of the board is filled with the same symbol' do
      game.board_game.board = %w[O 2 3 O 5 6 O 8 9]
      expect(game.win?).to eql(true)
  end
```
Kindly apply for the rest of the `it` blocks.
check the [link](https://github.com/rubocop-hq/rspec-style-guide#empty-lines-between-describes) for more info........